### ISSUE TITLE
Week 21: 70% demo and research into improving machine learning results

### ISSUE THREAD

**Issue by ealouie** [gw-sd-2016/NewsTextAnalysis#6]
@poorvi-vora

This week I made some updates to my program for the 70% demo. This included adding hyperlinks and navigation to the browser when they are clicked (f1b708cfcda6d50758a9b5b458bade16ed76852c). I originally tried to use Java's WebView to view the articles within the application, but I could not quite get that to work, so if I have any leftover time I might go back to that. I also made some formatting changes and deleted the machine learning (.arff and .csv) files on close of the program (65e8a31f39f3ebfe76c1197b0a3a238391b96c83).

The second half of the week after the demo I spent researching ways that I could make my model smarter and perform better dynamically over time. While I'm currently using the J48 algorithm, I found that the Naive Bayes algorithm can be trained incrementally. I still need to do some more research this week to figure out if this will work for me, more specifically, if this will allow me to update my model even after it has been created and used OR if it is just incremental to save space while creating one model with a training set that is very large.

**Comment by poorvi-vora** [comment]
The ability to use an incremental ML algorithm will be really helpful to you.

**Comment by ealouie** [comment]
@poorvi-vora Yeah, I'll be meeting with Professor Simha probably on Wednesday to discuss this more.
